common-lib/src/main/java/com/google/gct/idea/util/LandingPages.java

```diff
@@ -1,4 +1,4 @@
-package com.google.gct.login;
```

Comment:
my hesitation here is that the auth LandingPages really feel specific to the Account plugin. 

I think I introduced the IntelliJPlatform enum but I'm wondering if it's punching its weight here.  Maybe we'd simplify the dependencies a bit by just using PlatformUtils directly.